src/Api/Service/Sf.php

```diff
@@ -558,6 +558,18 @@ public function lex_stats($projectCode)
         throw new UserUnauthorizedException("User $this->userId is not a member of project $projectCode");
 	}
 
+	public function lex_stats_all($projectCode)
+	{
+		$projectModel = ProjectModel::getByProjectCode($projectCode);
+        $user = new UserModel($this->userId);
+
+        if ($user->isMemberOfProject($projectModel->id->asString())) {
+            return LexDbeDto::encode($projectModel->id->asString(), $this->userId, 1);
```

Comment:
So, on later inspection of this code, I'm not really clear how passing 1 into the lastFetchTime achieves more than 1000 entries at a time... maybe it does for you but I actually don't understand how it works.

Reply:
ok, yeah that's definitely a hack.  I want to make a proper stats endpoint to support what you are doing with the dashboard.  That will be a separate card.